### PR TITLE
Test swagger changes.

### PR DESCRIPTION
**Pull request by owenhuynMSFT**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Encoding.json [hunk 1]
```diff
@@ -1242,7 +1129,7 @@
         }
       },
       "type": "object",
-      "description": "Describes a list of inputs to a Job."
+      "description": "Describes a list of of inputs to a Job."
```
Comment (fengzhou-msft) [inline specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Encoding.json]: I'm seeing a few typos, such as `of of`, `the the`, `modifed`, are they intentional?

File: specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/AccountFilters.json [hunk 2]
```diff
@@ -199,6 +198,18 @@
...
+          "readOnly": true
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true
+        },
```
Comment (sanjaiganesh) [inline specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/AccountFilters.json]: these (name, id , type) must be part of the common definition for ProxyResource

File: specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/AccountFilters.json [hunk 3]
```diff
@@ -8,7 +8,6 @@
     "description": "This Swagger was generated by the API Framework.",
     "version": "2018-07-01"
   },
-  "host": "management.azure.com",
   "consumes": [
```
Comment (sanjaiganesh) [inline specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/AccountFilters.json]: >consumes [](start = 3, length = 8)

what is the reason for removing 'host' ?

Comment (sanjaiganesh) [reply specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/AccountFilters.json]: IMO it is required

---
In reply to: [387329551](https://github.com/Azure/azure-rest-api-specs/pull/8297#discussion_r387329551) [](ancestors = 387329551)

Comment (sanjaiganesh) [reply specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/AccountFilters.json]: same feedback for other files

---
In reply to: [387329591](https://github.com/Azure/azure-rest-api-specs/pull/8297#discussion_r387329591) [](ancestors = 387329591,387329551)

File: specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/AccountFilters.json (hunk 2 above)
Comment (sanjaiganesh) [reply specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/AccountFilters.json]: same feedback for other files

---
In reply to: [387329390](https://github.com/Azure/azure-rest-api-specs/pull/8297#discussion_r387329390) [](ancestors = 387329390)

**Comment by ravbhatnagar** [comment]
no response from PR owner on initial comments from ARM review. removing ARM review label. please re-add in case this PR is still relevant.

**Comment by fengzhou-msft** [comment]
Close this old PR. Feel free to reopen it or open another PR if you need to continue the work.

**Pull request closed without merging**